### PR TITLE
fix: resolve relative path when deleting

### PR DESCRIPTION
**Pull request by george-e-shaw-iv**
## What this PR does / why we need it

**JIRA ID**: XX-XX

### CODE REVIEW

File: cmd/devenv/apps/delete/delete.go [hunk 1]
```diff
@@ -97,22 +96,5 @@ func (o *Options) Run(ctx context.Context) error {
...
-		if a.Name == o.App {
-			found = true
-			break
-		}
-	}
-	if !found {
-		return fmt.Errorf("Failed to find application named '%s' that was deployed", o.App)
-	}
-
 	return app.Delete(ctx, o.log, o.k, b, o.conf, o.App, kr.GetConfig())
```
Comment (plisy) [inline cmd/devenv/apps/delete/delete.go:99]: Should the error from this be wrapped?

**Review by plisy — APPROVED** [review]

**Comment by george-e-shaw-iv** [comment]
```
 » /Users/georgeshaw/go/src/github.com/getoutreach/devenv/bin/devenv --skip-update apps del
ete .
INFO[0000] Deleting application from devenv...           app.name=reactor app.type=bootstrap
NAME                                          CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%
gke-loft-dev-us-west1-primary-99abb1c0-2j19   925m         5%     4514Mi          13%
gke-loft-dev-us-west1-primary-a9fe79da-6xnv   846m         5%     4461Mi          13%
gke-loft-dev-us-west1-primary-a9fe79da-7shv   616m         3%     2819Mi          8%
gke-loft-dev-us-west1-primary-a9fe79da-bbk0   1347m        8%     5257Mi          16%
gke-loft-dev-us-west1-primary-a9fe79da-l2vv   886m         5%     4372Mi          13%
gke-loft-dev-us-west1-primary-d98c7cbc-40w8   1858m        11%    1819Mi          5%
gke-loft-dev-us-west1-primary-99abb1c0-t7j3   <unknown>                           <unknown>               <unknown>               <unknown>
gke-loft-dev-us-west1-primary-d98c7cbc-kz1b   <unknown>                           <unknown>               <unknown>               <unknown>
gke-loft-dev-us-west1-primary-d98c7cbc-b4bc   <unknown>                           <unknown>               <unknown>               <unknown>
gke-loft-dev-us-west1-primary-99abb1c0-sgs6   <unknown>                           <unknown>               <unknown>               <unknown>
Error: devenv doesn't appear to be in a running state, run 'devenv status' for more information

ERRO[0006] failed to run: failed to delete application: exit status 1
```

wat

**Comment by jaredallard** [comment]
> ```
>  » /Users/georgeshaw/go/src/github.com/getoutreach/devenv/bin/devenv --skip-update apps del
> ete .
> INFO[0000] Deleting application from devenv...           app.name=reactor app.type=bootstrap
> NAME                                          CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%
> gke-loft-dev-us-west1-primary-99abb1c0-2j19   925m         5%     4514Mi          13%
> gke-loft-dev-us-west1-primary-a9fe79da-6xnv   846m         5%     4461Mi          13%
> gke-loft-dev-us-west1-primary-a9fe79da-7shv   616m         3%     2819Mi          8%
> gke-loft-dev-us-west1-primary-a9fe79da-bbk0   1347m        8%     5257Mi          16%
> gke-loft-dev-us-west1-primary-a9fe79da-l2vv   886m         5%     4372Mi          13%
> gke-loft-dev-us-west1-primary-d98c7cbc-40w8   1858m        11%    1819Mi          5%
> gke-loft-dev-us-west1-primary-99abb1c0-t7j3   <unknown>                           <unknown>               <unknown>               <unknown>
> gke-loft-dev-us-west1-primary-d98c7cbc-kz1b   <unknown>                           <unknown>               <unknown>               <unknown>
> gke-loft-dev-us-west1-primary-d98c7cbc-b4bc   <unknown>                           <unknown>               <unknown>               <unknown>
> gke-loft-dev-us-west1-primary-99abb1c0-sgs6   <unknown>                           <unknown>               <unknown>               <unknown>
> Error: devenv doesn't appear to be in a running state, run 'devenv status' for more information
> 
> ERRO[0006] failed to run: failed to delete application: exit status 1
> ```
> 
> wat

run `devenv status` for more information as it says 😛

**Comment by george-e-shaw-iv** [comment]
@jaredallard didn't tell me anything, seems to be a transient error. the next two runs succeeded.

part of me things we should also silence all gRPC output for all devenv commands the same way we did when running devenv status. i've been getting semi-consistent "noise"

**Pull request merged**